source/ts/server.ts

```diff
@@ -2,71 +2,74 @@ import net, {Socket} from 'net';
 import {EventEmitter} from 'events';
 
 export class Server extends EventEmitter {
-  server?: net.Server;
+  listenSocket?: net.Server;
```

Comment:
I thought this is a clearer name, rather than `Server` having a `server`

Reply:
<img width="455" alt="Screenshot 2023-12-12 at 16 12 21" src="https://github.com/FocusriteGroup/juce-end-to-end/assets/43950902/d4a709c1-3ad2-4cdd-9b3a-c33ab6f8d7ab">